engine/gamesys/src/gamesys/components/comp_mesh.cpp

```diff
@@ -780,39 +772,8 @@ namespace dmGameSystem
                 continue;
             }
```

Comment:
Since this data comes from the user, I would like us to make sure that the number of elements in the array is actually 6. Otherwise the usage below may go out of bounds.

Reply:
@JCash , how about also checking that the type is float ? 
Having an array of 6 8bit values will still get us out of bounds.

Reply:
True, it's a good point. Lets check that too then.

Reply:
It's already done in the commit bellow (910ce). 
Closing this conversation.